service/cziti/sdk.c

```diff
@@ -95,6 +95,10 @@ struct ziti_service_event* ziti_event_service_event(ziti_event_t *ev) {
     return &ev->event.service;
 }
 
+struct ziti_api_event* ziti_event_api_event(ziti_event_t *ev) {
```

Comment:
function simply casts and returns the correct union type to cgo else cgo thinks it's only a `ziti_event_t`